C/c4Base.cc

```diff
@@ -234,7 +234,6 @@ bool c4error_mayBeTransient(C4Error err) C4API {
     static CodeList kTransientWebSocket = {
         408, /* Request Timeout */
         429, /* Too Many Requests (RFC 6585) */
-        500, /* Internal Server Error */
```

Comment:
I wonder if sync gateway should return bad data error (400) instead of 500 for Invalid UTF-8 character?

Reply:
Maybe but that's far from the only exception the function can get.